nixos/modules/system/boot/tmp.nix

```diff
@@ -16,7 +16,7 @@ in
     boot.tmp = {
       cleanOnBoot = mkOption {
         type = types.bool;
-        default = false;
+        default = true;
```

Comment:
Shouldn't this change be documented in the release notes?

Reply:
I suppose, that's why I asked in the PR :sweat_smile:

Reply:
Yes it would require an entry in the release notes

Reply:
This is a change that requires to be brought to a wide audience of the Nixpkgs developer, e.g. Matrix channel, Discourse.